install/kubernetes/cilium/values.yaml

```diff
@@ -411,7 +413,7 @@ global:
     flowBufferSize: ~
     # Specifies the address the metric server listens to (e.g. ":12345"). The metric server is
     # disabled if this value is empty.
-    metricServer: ~
+    metricServer: ":6943"
```

Comment:
What is the intent of this change? Please do not enable the metrics server by default.

Reply:
Ok, I was actually moving the values from the hubble/values.yaml and this is the default value set in the hubble/values.yaml.
What do you think?

Reply:
For now I have set it back to empty as per your suggestion.